app_dart/test/request_handlers/vacuum_github_commits_test.dart

```diff
@@ -177,8 +180,7 @@ void main() {
         }
       };
       final Body body = await tester.get<Body>(handler);
-      expect(db.values.values.whereType<Commit>().length, 4 + Config.supportedRepos.length);
-      expect(db.values.values.whereType<Task>().length, 2 + (2 * 3 * Config.supportedRepos.length));
```

Comment:
I'm removing this assertion as it's not relevant to the test (it's testing the scheduler implementation which changes from time to time)